src/FSharpx.Collections/Collections.fs

```diff
@@ -234,26 +234,21 @@ module Seq =
 
     /// Extracts from a seq of Choice all the Choice1Of2 elements. All the Choice1Of2 elements are extracted in order.
     let inline choice1s xs =
-        let chooser = function
-                      | Choice1Of2 x -> Some x
-                      | _            -> None
+        let inline chooser a = match a with
```

Comment:
yes, it seemed that having true (un-inlined) functions here is not necessary and that inlining them could help performance. but I don't have any number to back it up.